Analyzer/plugins/Analyzer.cc

```diff
@@ -2881,23 +2880,22 @@ bool Analyzer::passPreselection(const reco::TrackRef track,
   passedCutsArray[10] = (  (typeMode_ != 5 && fabs(dxy) < globalMaxDXY_)
                         || (typeMode_ == 5 && fabs(dxy) < 4)) ? true : false;
   // Cut on the uncertainty of the pt measurement
-//  passedCutsArray[13] = (typeMode_ != 3 && (track->ptError() / track->pt()) < globalMaxPtErr_) ? true : false;
   passedCutsArray[11] = (typeMode_ != 3 && (track->ptError() / track->pt()) < pTerr_over_pT_etaBin(track->pt(), track->eta())) ? true : false;
   // Cut on the tracker based isolation
   passedCutsArray[12] = (!isMaterialTrack) ? true : false;
-//  passedCutsArray[13] = ( IsoTK_SumEt < globalMaxTIsol_) ? true : false;
+  passedCutsArray[13] = ( IsoTK_SumEt < globalMaxTIsol_) ? true : false;
```

Comment:
This will cut away quite a lot of signal

Reply:
Sure, but it's a test